src/20-bam/exp_builder.cc

```diff
@@ -89,8 +89,7 @@ exp_builder::exp_builder(exp_parser::notation const& postfix,
                  (*it == "%"))
           binary.reset(new bool_operation(*it));
         else
-          throw(exceptions::msg() << "unsupported operator " << *it
-                                  << " found while parsing expression");
+          throw exceptions::msg_fmt("unsupported operator {} found while parsing expression", *it);
```

Comment:
here I don"t understand how it can work. You don't have the namespace.
Please add a using namespcae com::centreon::exceptions;
and call directly msg_fmt